package.json

```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "body-parser": "^1.18.2",
+    "chai": "^4.1.2",
```

Comment:
I missed that the new line for `chai` was in the `dependencies` block. It is a _developer_ dependency (i.e., you don't need Chai to run GRNsight).